pkg/controller/handlers/task/task.go

```diff
@@ -215,6 +203,18 @@ func (t *Handler) UpdateTaskForEmail(thread *v1.Thread, req router.Request) erro
 			}
 			thread.Status.WorkflowNamesFromIntegration.EmailWorkflowName = workflow.Name
 		}
+
+		emailReceiver.Spec.Alias = workflow.Spec.Manifest.Alias
```

Comment:
This is the real fix: ensure that that workflow was created and set the alias here instead of above (where the alias on the workflow is actually blank).